src/domain/hooks/hooks.repository.ts

```diff
@@ -23,11 +28,19 @@ import { IStakingRepository } from '@/domain/staking/staking.repository.interfac
 import { memoize, MemoizedFunction } from 'lodash';
 
 @Injectable()
-export class HooksRepositoryWithNotifications implements IHooksRepository {
+export class HooksRepositoryWithNotifications
+  implements IHooksRepository, OnModuleInit, OnModuleDestroy
+{
   private static readonly HOOK_TYPE = 'hook';
+  private static readonly UNSUPPORTED_CHAIN_EVENT = 'unsupported_chain_event';
+  public static readonly UNSUPPORTED_EVENTS_LOG_INTERVAL = 60 * 1000; // 1 minute
```

Comment:
We could add this to the configuration if we plan to change it dynamically. I opted for keeping it here for now, but I'm open to externalise the configuration.